src/dsp_tools/commands/xmlupload/stash/upload_stashed_resptr_props.py

```diff
@@ -21,6 +22,7 @@ def upload_stashed_resptr_props(
 ) -> None:
     """
     After all resources are uploaded, the stashed resptr props must be applied to their resources in DSP.
+    Finally, the resource-ids should be removed if all their resptr props could be uploaded.
```

Comment:
removed from what?

Is this sentence relevant on this abstraction level? My impression is that this is a low-level detail that is relevant on the level of `_remove_empty_ids()`, but not on the level of `upload_stashed_resptr_props()`